cfgov/scripts/tests/test_smoke_tests.py

```diff
@@ -102,10 +134,17 @@ def test_allowed_timeouts(self, mock_get):
     def test_http_fail_connection_error(self, mock_get):
         mock_get.side_effect = requests.exceptions.ConnectionError
         http_smoke_test.check_urls('pro1')
-        self.assertEqual(mock_get.call_count, len(http_smoke_test.SHORT_RUN))
+        self.assertEqual(
+            mock_get.call_count,
+            len(http_smoke_test.SHORT_RUN)
+        )
 
     @mock.patch('scripts.http_smoke_test.requests.get')
-    def test_http_fail_request_error(self, mock_get):
+    @mock.patch('scripts.http_smoke_test.get_full_list')
```

Comment:
Looks like `mock_list` isn't needed for this one.

Reply:
True! Refactored.